lib/components/programs-list/index.tsx

```diff
@@ -0,0 +1,99 @@
+import { Flex, VStack, Heading, Text, Box, HStack, Link } from '@chakra-ui/react';
+import { ExternalLinkIcon } from '@chakra-ui/icons';
+
+const programs = [
+  {
+    title: 'LAUNCH PARTY',
+    description:
+      'Join OutstandingLife for a fun and informative conversation about how we can build an online community together.',
+    date: 'Friday, June 3',
+    time: '12:00pm - 1:00pm',
+    registerLink: 'https://tinyurl.com/2p9cfa3f',
+  },
+  {
+    title: 'BOOK CLUB',
+    description:
+      'Bob Linscott from the LGBTQIA+ Aging Project will facilitate a two-part discussion of Aging With Wisdom by Olivia Ames Hoblitzelle. Copies can be purchased online from Brookline Booksmith:',
+    descriptionLink: 'https://tinyurl.com/39bd2duk',
+    date: 'Tuesday, June 14 & 28',
+    time: '2:00pm - 4:00pm',
+    registerLink: 'https://tinyurl.com/7hkuzxb6',
+  },
+  {
+    title: "ONE SIZE DOESN'T SUIT ALL",
+    description:
+      'Older adults need multiple avenues for creating supportive communities, and safe environments, online and in-person, particularly LGBTQ+ older adults. Panel Discussion and Q&A with Paul Glass from LGBTQ Elders of Color and David Aronstein from OUTStandingLIFE.',
+    date: 'Wednesday, July 15',
+    time: '12:00pm - 1:30pm',
+    registerLink: 'https://tinyurl.com/yc267tzz',
+  },
+  {
+    title: 'MODERN PRONOUNS: THEY IS CORRECT',
+    description:
+      'Join Julie Nowak (she/they/xe), LGBTQ+ Initiative Coordinator at BayPath Elder Services, to learn about inclusive pronouns and affirming language.',
+    date: 'Thursday, June 16',
+    time: '2:00pm - 3:00pm',
+    registerLink: 'https://tinyurl.com/yc349dr9',
+  },
+];
+
+type ProgramFrontendProps = {
+  title: string;
+  description: string;
+  descriptionLink?: string;
+  date: string;
+  time: string;
+  registerLink: string;
+};
+
+const ProgramPreview: React.FC<ProgramFrontendProps> = ({
+  title,
+  description,
+  descriptionLink = '',
+  date,
+  time,
+  registerLink,
+}) => {
+  return (
+    <Flex flexDirection="row" gap={4} maxHeight="250px">
```

Comment:
You can make this responsive by making the flex direction "column" at the main breakpoint (see navbar for example)

Reply:
actually don't change anything for now